<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>789</width>
    <height>426</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Урок 07</string>
  </property>
  <property name="locale">
   <locale language="Russian" country="Russia"/>
  </property>
  <property name="inputMethodHints">
   <set>Qt::ImhNone</set>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>80</y>
      <width>151</width>
      <height>311</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <widget class="QPushButton" name="new_button">
       <property name="text">
        <string>Новый</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="open_button">
       <property name="locale">
        <locale language="Russian" country="Russia"/>
       </property>
       <property name="text">
        <string>Открыть</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="open_read_button">
       <property name="locale">
        <locale language="Russian" country="Russia"/>
       </property>
       <property name="text">
        <string>Открыть на чтение</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="save_button">
       <property name="locale">
        <locale language="Russian" country="Russia"/>
       </property>
       <property name="text">
        <string>Сохранить</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="print_button">
       <property name="locale">
        <locale language="Russian" country="Russia"/>
       </property>
       <property name="text">
        <string>Печать</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="settings_button">
       <property name="text">
        <string>Настройки</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="help_button">
       <property name="locale">
        <locale language="Russian" country="Russia"/>
       </property>
       <property name="text">
        <string>Справка</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>581</width>
      <height>351</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout"/>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>581</width>
      <height>32</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="left_button">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>40</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>Л</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="center_button">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>40</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>Ц</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="right_button">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>40</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>П</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="font_button">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>74</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>Шрифт</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="copy_format_button">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>163</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>Скопировать формат</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="apply_format_button">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>152</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>Применить формат</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>789</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="task_5_3_menu">
    <property name="locale">
     <locale language="Russian" country="Russia"/>
    </property>
    <property name="title">
     <string>Задание 5.3</string>
    </property>
    <addaction name="finder_action"/>
   </widget>
   <widget class="QMenu" name="file_menu">
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="open_action"/>
    <addaction name="open_rd_action"/>
    <addaction name="save_action"/>
    <addaction name="separator"/>
    <addaction name="print_action"/>
    <addaction name="separator"/>
    <addaction name="help_action"/>
   </widget>
   <widget class="QMenu" name="edit_menu">
    <property name="title">
     <string>Правка</string>
    </property>
    <addaction name="copy_action"/>
    <addaction name="cut_action"/>
    <addaction name="paste_action"/>
   </widget>
   <addaction name="file_menu"/>
   <addaction name="edit_menu"/>
   <addaction name="task_5_3_menu"/>
  </widget>
  <action name="finder_action">
   <property name="text">
    <string>Просмотрщик</string>
   </property>
  </action>
  <action name="open_action">
   <property name="text">
    <string>Открыть</string>
   </property>
  </action>
  <action name="open_rd_action">
   <property name="text">
    <string>Открыть на чтение</string>
   </property>
  </action>
  <action name="save_action">
   <property name="text">
    <string>Сохранить</string>
   </property>
  </action>
  <action name="help_action">
   <property name="text">
    <string>Справка</string>
   </property>
  </action>
  <action name="copy_action">
   <property name="text">
    <string>Копировать</string>
   </property>
  </action>
  <action name="cut_action">
   <property name="text">
    <string>Вырезать</string>
   </property>
  </action>
  <action name="paste_action">
   <property name="text">
    <string>Вставить</string>
   </property>
  </action>
  <action name="print_action">
   <property name="text">
    <string>Печать</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
